Source/Scene/ModelUtility.js

```diff
@@ -52,12 +52,15 @@ define([
         };
     };
 
-    ModelUtility.getAttributeOrUniformBySemantic = function(gltf, semantic) {
+    ModelUtility.getAttributeOrUniformBySemantic = function(gltf, semantic, programId) {
```

Comment:
Am I missing something here?  I don't see parameter `semantic` being used in this function.

Reply:
It's being used: `defined(parameter) && parameter.semantic === semantic`